prov/verbs/src/verbs_ep.c

```diff
@@ -412,7 +412,9 @@ static int fi_ibv_ep_enable(struct fid_ep *ep_fid)
 		}
 
 		attr.qp_type = IBV_QPT_RC;
-		attr.sq_sig_all = 1;
+		if ((ep->util_ep.tx_op_flags & FI_COMPLETION) && (ep->util_ep.tx_op_flags & FI_TRANSMIT)) {
+			attr.sq_sig_all = 1;
```

Comment:
Setting `sq_sig_all` to 0 requires to do SIGNALED operation after every `N` UNSIGNALED operations. As soon as Verbs doesn't have this logic, this fix breaks cases when only UNSIGNALED operations are used

Reply:
@dmitrygladkov I didn't quite understand the broken case. Sometimes, the sender side does not care when a message/RDMA update finished. If on signaled operations are used, it seems to me that the application just doesn't care when it's done.